src/harness/fake_sas.py

```diff
@@ -202,6 +202,30 @@ def GetEscSensorRecord(self, request, ssl_cert=None, ssl_key=None):
       # Return Empty if invalid Id
       return {}
 
+  def GetFullActivityDump(self, version, ssl_cert=None, ssl_key=None):
```

Comment:
This test case looks like copied from the content of FAD.1 PR. The below comments are related to those portion. Please use for reference:
1. The parameters ssl_cert=None, ssl_key=None arguments doesm't seem to be used
2. Please add some docstring description for this function to help understand the usage of this function.

Reply:
1. This is completely consistent with all of the other functions in this file. If this is wrong, let's address in a separate PR which cleans up all functions.
2. Not needed in fake_sas.py, the sas_interface.py contains the corresponding documentation.

Reply:
ok, we can do as what you suggested later.